src/mscorlib/src/System/Runtime/CompilerServices/AsyncMethodBuilder.cs

```diff
@@ -385,18 +385,6 @@ public void SetResult()
         public void SetException(Exception exception) { m_builder.SetException(exception); }
 
         /// <summary>
-        /// Called by the debugger to request notification when the first wait operation
-        /// (await, Wait, Result, etc.) on this builder's task completes.
-        /// </summary>
-        /// <param name="enabled">
-        /// true to enable notification; false to disable a previously set notification.
-        /// </param>
-        internal void SetNotificationForWaitCompletion(bool enabled)
```

Comment:
Thanks. Can you make sure these and others are added to model file so they don't get removed?